react/features/calendar-sync/reducer.js

```diff
@@ -21,7 +21,7 @@ const DEFAULT_STATE = {
     knownDomains: []
 };
 
-const MAX_DOMAIN_LIST_SIZE = 10;
+const MAX_DOMAIN_LIST_SIZE = 30;
```

Comment:
I think it's safe t have one, otherwise it may grow too big in extreme use cases and we don't have a flow to delete the entries. In non-extreme use cases this size will actually mean there is no size limit at all. What do you think?